.github/workflows/e2e-tests.yml

```diff
@@ -8,16 +8,37 @@ on:
     branches: [master]
     paths-ignore:
     - '**.md'
+  schedule:
+    - cron:  '0 */12 * * *'
 
 jobs:
+  create-matrix:
+    name: "Determine tests matrix"
+    runs-on: ubuntu-latest
+    timeout-minutes: 1  # 2021-02-03: Should only take a second.
+    outputs:
+      matrix: ${{ steps.create-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: create-matrix
+        run: |
+          MATRIX='[{"group":"pre-connection" },{"group":"connection" },{"group":"post-connection" }]'
```

Comment:
Just to make sure the behaviour I understand from this code is the intended one:

- On schedule all tests will run
- On pull request the gutenberg tests (with plugin set up) will not run

Reply:
Yes, but I'd frame it like this:

- For pull requests, we run all the tests with bundled Gutenberg.
- For the scheduled job, we run all the tests with bundled Gutenberg PLUS blocks tests with the latest dev Gutenberg